src/components/WalletConnect/ModalButton.js

```diff
@@ -11,20 +11,20 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     marginLeft: 12,
     borderWidth: 2,
-    borderColor: '#cecece',
+    borderColor: COLORS.borderColor,
   },
   buttonHighlight: {
-    backgroundColor: PRIMARY_COLOR,
+    backgroundColor: COLORS.primary,
     borderWidth: null,
     borderColor: null,
   },
   buttonText: {
-    color: '#808080',
+    color: COLORS.midContrastDetail,
```

Comment:
Change from `#808080` to `#9e9e9e`

Reply:
Why don't we keep the same color?

Reply:
Reverted to original color on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)